integration-tests/src/test/java/org/openmrs/module/webservices/rest/ITBase.java

```diff
@@ -27,7 +27,7 @@
 
 public abstract class ITBase {
 	
-	private static final Object serverStartupLock = new Object();
+	private static final Object SERVER_STARTUP_LOCK = new Object();
```

Comment:
@docjon09 are you sure `serverStartupLock` is a variable?

Reply:
I didn't knew objects aren't considered variables. I have updated it now. Thanks.